tests/Doctrine/Tests/ORM/Functional/ManyToManyBasicAssociationTest.php

```diff
@@ -309,8 +309,10 @@ public function addCmsUserGblancoWithGroups($groupCount = 1)
      */
     public function testUpdateDeleteSizeSubselectQueries()
     {
-        $this->_em->createQuery("DELETE Doctrine\Tests\Models\CMS\CmsUser u WHERE SIZE(u.groups) = 10")->execute();
-        $this->_em->createQuery("UPDATE Doctrine\Tests\Models\CMS\CmsUser u SET u.status = 'inactive' WHERE SIZE(u.groups) = 10")->execute();
+        $this->_em->createQuery("DELETE " . CmsUser::class . " u WHERE SIZE(u.groups) = 10")->execute();
```

Comment:
It happens to already be there:

https://github.com/doctrine/doctrine2/blob/22ecc2d58cdf1e8a9b932d99b952c94afae7e873/tests/Doctrine/Tests/ORM/Query/DeleteSqlGenerationTest.php#L257-L266
https://github.com/doctrine/doctrine2/blob/22ecc2d58cdf1e8a9b932d99b952c94afae7e873/tests/Doctrine/Tests/ORM/Query/UpdateSqlGenerationTest.php#L168-L177

Should we just remove the funcional test then?